handlers/istio_config.go

```diff
@@ -51,7 +51,7 @@ func IstioConfigList(w http.ResponseWriter, r *http.Request) {
 			defer wg.Done()
 			// We don't filter by objects when calling validations, because certain validations require fetching all types to get the correct errors
 			istioConfigValidationResults, errValidations := business.Validations.GetValidations(namespace, "")
-			if errValidations != nil && err == nil {
+			if errValidations != nil && *err == nil {
```

Comment:
@lucasponce @mwringe This and the other similar changes look like a bug.
Not sure if it is worth to cherry-pick these changes.

Reply:
I don't think it's worth cherry-picking as an hotfix, because anyway `err` will anyway never be `nil` so it's not going to panic due to potential dereference. Only problem is that `errValidations` is never going to be caught, I don't think it's an urgent fix.

But good catch anyway! I'm glad we can rely on a better linter soon :-)